internal/pkg/client/request.go

```diff
@@ -56,28 +56,38 @@ func NewRequest(id int, sender Sender, request *resty.Request) *Request {
 }
 
 func (r *Request) SetResult(result interface{}) *Request {
+	r.lock.Lock()
```

Comment:
Na Windows sa niekedy HTTP pool zaseklo.
Presiel som pouzitie zamkov, a skontroloval som, ze:
- je zamok na kazdom citani/zapisovani hodnoty
- zamok nie je na ziadnej blokujucej operacii
- v teste sa mi to nepodarilo nasimulovat, existujuce testy prechadzaju (je ich tam celkom dost `pool_test.go`)